api/routes/resource.routes.js

```diff
@@ -84,5 +162,9 @@ module.exports = (app) => {
 
 let userIsResourceOwner = async (user, fields) => {
   let resource = await resourceUtil.getById(fields._id);
-  return resource.user._id == user._id;
+  if (resource?.user != undefined) {
+    return resource.user._id == user._id;
+  } else {
+    return false;
+  }
```

Comment:
`return resource?.user._id == user._id` would work since the ? does the undefined check.

Reply:
would it be` resource.user?._id` since it's the user that's sometime undefined

Reply:
yup